extra-rocm/rocm-opencl-runtime/autobuild/prepare

```diff
@@ -0,0 +1,7 @@
+acbs_copy_git
+
+abinfo "Building ROCclr used for ROCm-OpenCL-Runtime..."
+cd ${SRCDIR}/../ROCclr
```

Comment:
You could have saved the last `cd ${SRCDIR}` if you used...

```
(
    cd "$SRCDIR"/../ROCclr
    ...
)
```

Also, please quote `$SRCDIR`.